.agent/implementation_plans/org_feature.md

```
@@ -0,0 +1,168 @@
+# 組織（Organization）機能 実装計画
+
+## データモデル
+
+### 新規コレクション
+
+#### `users_v0/{userId}`
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)